IncidentFields/incidentfield-emailclassification.json

```diff
@@ -0,0 +1,44 @@
+{
+    "id": "incident_emailclassification",
+    "version": -1,
+    "modified": "2019-09-02T12:32:29.881674101Z",
+    "name": "Email Classification",
+    "ownerOnly": false,
+    "placeholder": "",
+    "description": "The category of the email.",
+    "cliName": "emailclassification",
+    "type": "singleSelect",
+    "closeForm": true,
+    "editForm": true,
+    "required": false,
+    "script": "",
+    "fieldCalcScript": "",
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "selectValues": [
+        "",

```

Comment:
why empty value here?

Reply:
This is how its exported if you choose not to have any value as default